bbot/core/helpers/cache.py

```diff
@@ -22,10 +22,12 @@ def cache_get(self, key, text=True, cache_hrs=24 * 7):
             else:
                 open_kwargs["mode"] = "rb"
             log.debug(f'Using cached content for "{key}"')
-            return open(filename, **open_kwargs).read()
+            # Using a  block to ensure the file is closed automatically
+            with open(filename, **open_kwargs) as f:
+                return f.read()
         else:
             log.debug(f'Cached content for "{key}" is older than {cache_hrs:,} hours')
-
+    return None
```

Comment:
https://docs.astral.sh/ruff/rules/unnecessary-return-none

Reply:
thanks for enlightenment 😀